x/finality/types/finality.go

```diff
@@ -191,3 +191,19 @@ func (e *Evidence) ExtractBTCSK() (*btcec.PrivateKey, error) {
 		e.forkMsgToSign(e.SigningContext), e.ForkFinalitySig.ToModNScalar(), // msg and sig for fork block
 	)
 }
+
+func (fp *FinalityProviderDistInfo) Status() btcstktypes.FinalityProviderStatus {
```

Comment:
I am not very confident in defining the previous state like this, it might be usefull to use in the migration if we decide to store the previous state in the